### DISCUSSION TITLE
HMAC-SHA256 support

### DISCUSSION THREAD

**Discussion by bruno-robert** [lepture/authlib#363] [Q&A] [answered]
I was wondering if authlib has support for the signature method: HMAC-SHA256? 
I know the default is HMAC-SHA1.

I'm also wondering, how hard would it be to implement HMAC-SHA256 if it currently isn't supported?

**Comment by lepture** [comment, marked as answer]
https://docs.authlib.org/en/latest/flask/1/customize.html

**Reply by bruno-robert** [reply]
yup, I missed that, thank you!
